vendor/engines/secure_rooms/app/views/secure_rooms/card_readers/new.html.haml

```diff
@@ -0,0 +1,15 @@
+= content_for :h1 do
+  = current_facility
+= content_for :sidebar do
+  = render "admin/shared/sidenav_product", sidenav_tab: "secure_rooms"
+= content_for :tabnav do
+  = render "admin/shared/tabnav_product", secondary_tab: "card_readers"
+
+%h2= @product
+%h3= text("views.admin.secure_rooms.card_readers.new.title")
+
+= simple_form_for @card_reader, url: facility_secure_room_card_readers_path(current_facility, @product) do |f|
```

Comment:
Rails is smart enough that you should be able to (and I just tested this out) to use `= simple_form_for [current_facility, @product, @card_reader] do |f|` for both new and edit. It switches the URL based on if the card_reader is persisted or not.

That might also lead to some more sharing between the new and edit partials. At the very least, the input part should be shared. We sometimes do that in a `_form.html.haml` partial.